File: Examples/Outlook/VB/Examples/OutlookExamples/FormMain.vb
Public Class FormMain

    Public Sub New()

        InitializeComponent()

        Me.Text = "NetOffice Outlook Examples in VB"
        LoadExamples()

    End Sub

    Private Sub LoadExamples()

        LoadExample(New Example01())
        LoadExample(New Example02())
        LoadExample(New Example03())
        LoadExample(New Example04())
        LoadExample(New Example05())
        LoadExample(New Example06())
        LoadExample(New Example07())

    End Sub

End Class

File: Examples/Outlook/C#/Standard Examples/OutookExamples/FormMain.cs
using System;
using System.Windows.Forms;
using ExampleBase;

namespace OutlookExamplesCS4
{
    public partial class FormMain : ExampleForm
    {
        public FormMain()
        {
            InitializeComponent();
            this.Text = "NetOffice Outlook Examples in C#";
            LoadExamples();
        }

        private void LoadExamples()
        {
            LoadExample(new Example01());
            LoadExample(new Example02());
            LoadExample(new Example03());
            LoadExample(new Example04());
            LoadExample(new Example05());
            LoadExample(new Example06());
            LoadExample(new Example07());
        }
    }
}
